src/ur_arm.cpp

```diff
@@ -358,60 +377,73 @@ void UR5eArm::move_through_joint_positions(const std::vector<std::vector<double>
     }
 }
 
-pose UR5eArm::get_end_position(const ProtoStruct&) {
+pose URArm::get_end_position(const ProtoStruct&) {
     const std::lock_guard<std::mutex> guard{current_state_->mu};
     std::unique_ptr<rtde_interface::DataPackage> data_pkg = current_state_->driver->getDataPackage();
     if (data_pkg == nullptr) {
-        VIAM_SDK_LOG(warn) << "UR5eArm::get_end_position got nullptr from driver->getDataPackage()";
+        VIAM_SDK_LOG(warn) << "URArm::get_end_position got nullptr from driver->getDataPackage()";
         return pose();
     }
     if (!data_pkg->getData("actual_TCP_pose", current_state_->tcp_state)) {
-        VIAM_SDK_LOG(warn) << "UR5eArm::get_end_position driver->getDataPackage().getData(\"actual_TCP_pos\") returned false";
+        VIAM_SDK_LOG(warn) << "URArm::get_end_position driver->getDataPackage().getData(\"actual_TCP_pos\") returned false";
         return pose();
     }
     return ur_vector_to_pose(current_state_->tcp_state);
 }
 
-bool UR5eArm::is_moving() {
+bool URArm::is_moving() {
     return current_state_->trajectory_running.load();
 }
 
-UR5eArm::KinematicsData UR5eArm::get_kinematics(const ProtoStruct&) {
+URArm::KinematicsData URArm::get_kinematics(const ProtoStruct&) {
+    // The `Model` class absurdly lacks accessors
+    const std::string model_string = [&] {
+        const auto family = ModelFamily{"viam", "universal-robots"};
```

Comment:
whats the rationale for not making this a const on URArm? we only need it here and registration currently?

Reply:
It could be. It'd be nicer if ModelFamily had a constexpr ctor, but it doesn't. But I can make a static function on URArm to return that ModelFamily, and then I can also add a static `model` function. That'd get rid of some of the duplication fairly nicely.

Reply:
Done